### PR TITLE
Brands API Reference update for removePoweredByOkta field addition

### PR DESCRIPTION
**Pull request by albertchen-okta**
## Description:
- **What's changed?** 
- Updated Brands API to reflect new feature to turn off "powered by okta" branding on sign-in page and Okta End-user dashboard

- **Is this PR related to a Monolith release?**
- 2021.10.0

### Resolves:

*https://oktainc.atlassian.net/browse/OKTA-430577 and https://oktainc.atlassian.net/browse/OKTA-424736

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 1]
```diff
@@ -222,7 +227,7 @@ Content-Type: application/json
 }
 ```
 
-The following example resets a privacy policy URL to use the Okta default privacy policy URL.
+The following example resets a privacy policy URL to use the Okta default privacy policy URL and remove Powered by Okta from all pages.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:230]:
```suggestion
The following example resets a privacy policy URL to use the Okta default privacy policy URL and removes the Powered by Okta label from all pages.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 2]
```diff
@@ -1121,22 +1129,33 @@ Resets a privacy policy URL
 }
 ```
 
+Updates setting to remove Powered by Okta
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1132]:
```suggestion
Updates setting to remove the Powered by Okta wording
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 3]
```diff
@@ -1121,22 +1129,33 @@ Resets a privacy policy URL
...
 
-| Property                  | Type                    | Description                     |
-| ------------------------- | ----------------------- | ------------------------------- |
-| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL       |
-| `id`                      | String                  | Brand ID                        |
-| `_links`                  | [Links](#links-object)  | Link relations for this object  |
+| Property                  | Type                    | Description                                             |
+| ------------------------- | ----------------------- | ------------------------------------------------------- |
+| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL                               |
+| `removePoweredByOkta`     | boolean                 | Setting to remove "Powered by Okta" from the experience |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1148]:
```suggestion
| `removePoweredByOkta`     | boolean                 | Setting to remove the  "Powered by Okta" wording from the experience |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 4]
```diff
@@ -1102,6 +1109,7 @@ The Brand request object defines the following properties:
 | ----------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------- |
 | `agreeToCustomPrivacyPolicy`  | boolean                 | (Optional) Consent for updating the custom privacy policy URL. Not required when resetting the URL. |
 | `customPrivacyPolicyUrl`      | String                  | Custom privacy policy URL. Default value: `null`.                                                   |
+| `removePoweredByOkta`         | boolean                 | (Optional) Flag to remove "Powered by Okta" from the experience. Defaulted to `false`               |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1112]:
```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Flag that removes the "Powered by Okta" wording from the experience. Defaults to `false`               |
```

**Review by susanharper-okta — COMMENTED** [review]
couple comments

**Review by DanMacDonald-okta — APPROVED** [review]
🚀

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 5]
```diff
@@ -1102,6 +1109,7 @@ The Brand request object defines the following properties:
 | ----------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------- |
 | `agreeToCustomPrivacyPolicy`  | boolean                 | (Optional) Consent for updating the custom privacy policy URL. Not required when resetting the URL. |
 | `customPrivacyPolicyUrl`      | String                  | Custom privacy policy URL. Default value: `null`.                                                   |
+| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard.. Defaulted to `false`        |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1112]:
```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false` .       |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 6]
```diff
@@ -1121,22 +1129,33 @@ Resets a privacy policy URL
...
 
-| Property                  | Type                    | Description                     |
-| ------------------------- | ----------------------- | ------------------------------- |
-| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL       |
-| `id`                      | String                  | Brand ID                        |
-| `_links`                  | [Links](#links-object)  | Link relations for this object  |
+| Property                  | Type                    | Description                                             |
+| ------------------------- | ----------------------- | ------------------------------------------------------- |
+| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL                               |
+| `removePoweredByOkta`     | boolean                 | Remove "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1148]:
```suggestion
| `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```

**Review by susanharper-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 7]
```diff
@@ -1102,6 +1109,7 @@ The Brand request object defines the following properties:
 | ----------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------- |
 | `agreeToCustomPrivacyPolicy`  | boolean                 | (Optional) Consent for updating the custom privacy policy URL. Not required when resetting the URL. |
 | `customPrivacyPolicyUrl`      | String                  | Custom privacy policy URL. Default value: `null`.                                                   |
+| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false` .       |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1112]:
```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false`.       |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 8]
```diff
@@ -1121,22 +1129,33 @@ Resets a privacy policy URL
...
 
-| Property                  | Type                    | Description                     |
-| ------------------------- | ----------------------- | ------------------------------- |
-| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL       |
-| `id`                      | String                  | Brand ID                        |
-| `_links`                  | [Links](#links-object)  | Link relations for this object  |
+| Property                  | Type                    | Description                                             |
+| ------------------------- | ----------------------- | ------------------------------------------------------- |
+| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL                               |
+| `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1148]:
```suggestion
| `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```

**Pull request merged**